Add release notes for 0.14.2 (#890)
submariner-io/submariner-website: src/content/community/releases/_index.en.md

weight = 40
+++
<!-- markdownlint-disable no-duplicate-header -->
## v0.14.2

This is a bugfix release:

* Fix issues in Azure cloud prepare:

>   * Machine set name now changed to region + uuid and limited to 20 characters to prevent issues long cluster name.

Reviewer: ```suggestion
  * Machine set names are now based on region + uuid and limited to 20 characters to prevent issues with long cluster names.
```

>   * Machine set name now changed to region + uuid and limited to 20 characters to prevent issues long cluster name.

Reviewer 2: ```suggestion
  * Machine set name convention was changed to region + uuid and is now limited to 20 characters to prevent issues seen with long cluster names.
```

>   * Machine set creation and deletion logic was updated to prevent creation of multiple g/w nodes.

Reviewer: ```suggestion
  * Machine set creation and deletion logic was updated to prevent creation of multiple gateway nodes.
```

>   * Machine set creation and deletion logic was updated to prevent creation of multiple g/w nodes.

Reviewer 2: ```suggestion
  * Machine set creation and deletion logic was updated to prevent creation of multiple gateway nodes.
```

>   * Image name is now obtain from existing Machine set.

Reviewer: ```suggestion
  * Image names are now retrieved from existing machine sets.
```

>   * Image name is now obtain from existing Machine set.

Reviewer 2: ```suggestion
  * Image name is now obtained from existing machine set.
```

> * Fix e2e ext-net socket permission denied error.

Reviewer: ```suggestion
* Fix a socket permission denied error in external network end-to-end tests.
```

Is this actually visible to end users?

* The `subctl gather` command will now gather iptables logs for Calico and kindnet CNIs.

> * Stop calling cloud prepare with empty ports list to avoid errors.

Reviewer: ```suggestion
* The `cloud prepare` command no longer causes errors if the list of ports is empty.
```

> * Fix issue when `subctl cloud` commands was not always return status.

Reviewer: ```suggestion
* `subctl cloud` commands now always return the appropriate status.
```

> * Fix issue when `subctl cloud` commands was not always return status.

Reviewer 2: ```suggestion
* Fix issue when `subctl cloud` commands were not always return status.
```

> * The `subctl` operations which deploys images are now able to override those images. The overrides are specified using `--image-override`:

Reviewer: ```suggestion
* `subctl` operations which deploy images now allow those images to be overridden. The overrides are specified using `--image-override`:
```

  * `subctl benchmark`.
  * `subctl verify`.

>   * `subctl diagnose` sub-commands which deploy images.

Reviewer 2: ```suggestion
  * `subctl diagnose`.
```

* Fix issues in Service Discovery:
  * Namespace is now included in EndpointSlice name.

>   * Look up service EndpointSlice is now done by labels.

Reviewer: ```suggestion
  * EndpointSlice service lookup is now done using labels.
```
